libs/ledger-live-common/src/families/evm/adapters.ts

```diff
@@ -51,48 +53,92 @@ export const transactionToEthersTransaction = (
  */
 export const etherscanOperationToOperation = (
   accountId: string,
-  address: string,
   tx: EtherscanOperation
-): Operation | null => {
+): Operation => {
+  const { xpubOrAddress: address } = decodeAccountId(accountId);
   const from = eip55.encode(tx.from);
   const to = tx.to ? eip55.encode(tx.to) : "";
   const value = new BigNumber(tx.value);
   const fee = new BigNumber(tx.gasUsed).times(new BigNumber(tx.gasPrice));
 
   const type = ((): OperationType => {
-    // Do not show interaction with Smart Contract
-    if (tx.contractAddress) {
-      return "NONE";
-    }
-
     if (to === eip55.encode(address)) {
       return "IN";
     }
     if (from === eip55.encode(address)) {
+      return value.eq(0) ? "FEES" : "OUT";
+    }
+
+    return "NONE";
+  })();
+
+  return {
+    id: encodeOperationId(accountId, tx.hash, type),
+    hash: tx.hash,
+    type: type,
+    value: type === "OUT" || type === "FEES" ? value.plus(fee) : value,
```

Comment:
I try to avoid `[].includes()` when possible because I find it hard to read sometimes. But I'll use it for sure if we add another value later :ok_thumb: